.travis.yml

```diff
@@ -28,6 +28,10 @@ script:
   # prin the status of the services
   - npm run service-status
 
+  # show the daos that are indexed
+  - curl 'http://127.0.0.1:8000/subgraphs/name/daostack' -H 'Origin: http://127.0.0.1:8000' -H 'Accept-Encoding: gzip, deflate, br' -H 'Accept-Language: nl,it;q=0.9,en-US;q=0.8,en;q=0.7,de;q=0.6,pt;q=0.5,es;q=0.4' -H 'User-Agent: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/72.0.3626.96 Safari/537.36' -H 'Content-Type: application/json' -H 'Accept: application/json' -H 'Referer: http://127.0.0.1:8000/subgraphs/name/daostack/graphql' -H 'Cookie: seen_disclaimer=true; seen_tour=true; io=9TnmcVwp4hrE5zgPAAAB' -H 'Connection: keep-alive' --data-binary '{"query":"  {daos { id }}","variables":null,"operationName":null}' --compressed
```

Comment:
fyi, you don't really need all the headers... the subgraph will accept just the url & query ^_^...
makes it a little more readable :)